package.json

```diff
@@ -42,23 +48,58 @@
     "oidc-client": "^1.4.1"
   },
   "devDependencies": {
-    "@types/chai": "^4.0.5",
-    "@types/mocha": "^2.2.44",
-    "@types/sinon": "^4.0.0",
-    "chai": "^4.1.2",
-    "cpx": "^1.5.0",
-    "karma": "^1.7.1",
-    "karma-cli": "^1.0.1",
-    "karma-mocha": "^1.3.0",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-typescript": "^3.0.8",
-    "mocha": "^4.0.1",
-    "phantomjs": "^2.1.7",
-    "proxyquire": "^1.8.0",
-    "sinon": "^4.1.2",
-    "tslint": "^5.8.0",
-    "tslint-microsoft-contrib": "^5.0.1",
+    "@types/jasmine": "^2.5.53",
+    "@types/node": "^7.0.11",
```

Comment:
Note-to-self: Test this. Does using `Jasmine` instead of `Mocha/Chai/Sinon` just work?

Reply:
The test are not tested 😄

Reply:
tests are working now